libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md

```diff
@@ -0,0 +1,74 @@
+# Sierra-F1 Autopilot by Sierra Aerospace
+
+## Features
+# Main Board:
+ - STM32H743 microcontroller
+ - 32MB QSPI flash
+ - BMI088+ICM42688 IMUs 
+ - DPS310 barometer on SPI 
+ - SDMMC support
+ - 4 UARTs- 2 with HW flow control
+ - 6 PWM outputs with DSHOT capability
+ - Dual FDCAN
+ - RCIN port
+ - Dual power input with 2xI2C
+
+# Daughter board:
+ -Ethernet PHY
+ -ICM42688/ICM45686/ADIS16507
+ -DPS310
+ -More PWMs with ALT function
```

Comment:
what ALT function? need more info for setup and capabilities...how many PWMs,groups,etc.? if this has a co-processor doesn't it need to be specified somehow in hwdef?

Reply:
```suggestion
 -More PWMs with ALT function
```
```suggestion
 -More PWMs with ALT function (this is a future planned addition, not present now)
```